xbmc/addons/kodi-addon-dev-kit/include/kodi/kodi_vfs_types.h

```diff
@@ -58,6 +58,7 @@ extern "C"
     const char* password;
     const char* redacted;
     const char* sharename;
+    const char* protocol;
```

Comment:
No need for this change as the header is not used anymore and should have been removed already.

Reply:
@ksooo : But either I leave it here for consistency / to prevent confusion or I'll PR the removal rightaway. What do you prefer?

Reply:
@ksooo As long it is there, better to change with others, confusion can then come

Reply:
Okay. I have no hard feelings about that.